CombinatorialGames/SignExpansion/Small.lean

```diff
@@ -0,0 +1,112 @@
+/-
+Copyright (c) 2025 Aaron Liu. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Aaron Liu
+-/
+import CombinatorialGames.SignExpansion.Basic
+
+/-!
+# Small sign expansions
```

Comment:
I'd also rather have this file in a separate PR. Every PR should focus on a single thing!